Makefile

```diff
@@ -408,6 +408,9 @@ bin/netlisten: tools/cmd/netlisten/* tools/go.sum tools/pkg/*
 	@mkdir -p bin
 	cd tools && go build -o ../bin/netlisten ./cmd/netlisten
 
+# isopatch injects files into an ISO with placeholders without rebuilding it.
```

Comment:
using `it` to refer to the ISO and then the next sentence starts with `it` referencing isopatch is a bit confusing.

maybe `# isopatch injects ... without rebuilding the ISO."